src/pybamm_cookiecutter/cli.py

```diff
@@ -0,0 +1,40 @@
+import copier
+import argparse
+from pathlib import Path
+from colorama import Fore
+import os
```

Comment:
Would we need to add `colorama` as a dependency?

Reply:
If we are using it in a file, we should still list it in our dependencies since we can't guarantee whether `copier` will continue to depend on it.